src/test/java/org/openmrs/contrib/qaframework/automation/VitalsAndTriagingSteps.java

```diff
@@ -148,6 +151,42 @@ public void savePatientVitals() {
 
 	@Then("the system adds patient vitals into the vitals table")
 	public void systemAddsPatientVitals() {
-		assertTrue(patientCaptureVitalsPage.containsText("Vitals"));
+		assertNotNull(visitsDashboardPage.getEncountersCount());
+	}
+	
+	@When("a user clicks on edit vitals icon from patient visits dashboard")
+	public void loadEditVitalsPage() {
+		editVitalsPage = (EditVitalsPage) visitsDashboardPage
+				.goToEditVitalsPage().waitForPage();
```

Comment:
Can this be on the same line , @kdaud  seems to have resolved this line code format breaks

Reply:
@sherrif10 that's ok if I have to put it on the same line. However, doesn't the line appear so long that the code can get out of sight of the developer? :)

Reply:
I second @sherrif10 idea of putting the stuff on the same line to have code sanity within the module